hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -2243,6 +2321,7 @@ int SaraNcpClient::interveneRegistration() {
                 psd_.reset();
                 registrationInterventions_++;
                 CHECK_PARSER_OK(setModuleFunctionality(CellularFunctionality::MINIMUM));
+                CHECK_PARSER_OK(setOperationMode(CellularOperationMode::PS_ONLY, true /* check */));
```

Comment:
This is in the block checking for 
```
    if (ncpId() != PLATFORM_NCP_SARA_R410 && ncpId() != PLATFORM_NCP_SARA_R510) {
```
So presumably this is for SARA U260? Do we want to send `AT+CEMODE` to non 410/510 modems?

Reply:
At this time we are only targeting R510 based devices, so yeah that should be removed.